### DISCUSSION TITLE
How can I use a one-off breakpoint

### DISCUSSION THREAD

**Discussion by avaltat** [tailwindlabs/tailwindcss#10420] [Help]
Hello, and first thank you for the amazing work you have done with tailwind. 

How can I use a one-off breakpoint with tailwind. I tried this https://tailwindcss.com/docs/responsive-design#arbitrary-values, but this don't seem to work (see this sandbox https://codesandbox.io/s/vigorous-northcutt-v7ssdm?file=/src/App.tsx).

Is there something I don't get from the tailwind documentation ? Is this a bug ?

Thank you.

**Comment by wongjn** [comment]
I'm not sure how you're pulling in Tailwind in the sandbox, but the syntax is correct and works [in Tailwind Play](https://play.tailwindcss.com/pqBYfuIhGN).

**Reply by avaltat** [reply]
ok thank you

**Comment by aenial51** [comment]
My problem has been solved, I'm delighted, the problem came from the decoder for me so they changed it to me. And today everything is working very well. 
[www.connectebt.com
](https://www.connectebt.us/)
